cores/cv32e40p/user_manual/source/control_status_registers.rst

```diff
@@ -987,22 +1030,20 @@ Table 14: MHARTID
   CSR Address: 0x005
 
   +-------------+-----------+---------------------------------------------------------------------------------------------------------------+
-  |   Bit #     |   R/W     |   Description                                                                                                 |
+  |   Bit #     |   Mode    |   Description                                                                                                 |
   +=============+===========+===============================================================================================================+
-  | 31 : 2      |   R/W     | BASE: The trap-handler base address, always aligned to 256 bytes, i.e., utvec[7:2] is always set to 0.        |
+  | 31 : 2      |   RW      | BASE: The trap-handler base address, always aligned to 256 bytes, i.e., utvec[7:2] is always set to 0.        |
   +-------------+-----------+---------------------------------------------------------------------------------------------------------------+
-  |  1 : 0      |   R/W     | MODE: 00 = direct mode, 01 = vectored mode. Writes to bit 1 are ignored and this bit always reads as 0.       |
+  |  1 : 0      |   RW      | MODE: 00 = direct mode, 01 = vectored mode. Writes to bit 1 are ignored and this bit always reads as 0.       |
```

Comment:
bit 1 is RO

Reply:
Yeah, I did not make any attempt to update the User Mode CSRs, but have made this correction.